test/update_command.bats

```diff
@@ -34,23 +34,23 @@ teardown() {
 }
 
 @test "asdf update should checkout the latest non-RC tag" {
-  local tag=$(git tag | grep -vi "rc" | head -1)
+  local tag=$(git tag | grep -vi "rc" | tail -1)
```

Comment:
Why the switch from head to tail? This shouldn't matter.

Reply:
@Stratus3D It doesn't really matter. I chose to do this because the test specifies `latest tag`  but `head` returns the first tag. 
I just wanted to ensure the test description matched the contents